### DISCUSSION TITLE
Amazing but I can't get it to work...

### DISCUSSION THREAD

**Discussion by deruiter** [khoih-prog/WiFiManager_NINA_Lite#17] [Q&A] [answered]
First thank you so much for making this!!  But sadly I'm not able after a few hours tinkering to get any further, getting it to work in any way.  

First I tried to use the Example for my Arduino 33IoT board (SAMD) but then this is the output:

`Starting SAMD_WiFiNINA on SAMD NANO_33_IOT
WiFiManager_NINA_Lite v1.4.1
[WN] NoWiFi
[WN] Hostname=SAMD-Master-Controller
Flag read = 0xffffffff
No doubleResetDetected
SetFlag write = 0xd0d01234
[WN] CCSum=0xe11e,RCSum=0xffffffff
[WN] ChkCrR:CrCCsum=0xaf50,CrRCsum=0xffffffff
[WN] InitCfgFile,sz=236
[WN] SaveEEPROM,Sz=1024,DataSz=0,WCSum=0xc7d
[WN] CrCCSum=0xc30
[WN] bg: isForcedConfigPortal = false
[WN] bg:Stay forever in CP:No ConfigDat`

I looked in the several .h files but I have absolutely no idea what to change to get it to work...  

Sadly I can't find anyone on the internet which has used this great library so I can't learn from others...

You tried so hard to docoment this but for me it's too complicated...

**Comment by khoih-prog** [comment, marked as answer]
Thanks for your interest and nice words about the library.

Even this is the simplest way to use `WiFiManager`, it also requires certain document readings to start, such as

1. [**So, how it works?**](https://github.com/khoih-prog/WiFiManager_NINA_Lite#so-how-it-works)
2. [**SAMD_WiFiNINA on Nano-33-IoT**](https://github.com/khoih-prog/WiFiManager_NINA_Lite#1-samd_wifinina-on-nano-33-iot)

You don't need to change any `.h` file yet to start a simple example, such as [**SAMD_WiFiNINA**](https://github.com/khoih-prog/WiFiManager_NINA_Lite/tree/master/examples/SAMD_WiFiNINA) which you correctly use.

I don't see any issue with your `terminal` post. You just need to open your computer/cell phone, connect to the SSID with the displayed password

```
[WN] SSID=WIFININA_51F485,PW=MyWIFININA_51F485
```

then connect the browser to `192.168.4.1` to display the Config Portal.

The document, which must be read if not familiar with the `WiFiManager`, is so complete compared to any other `WiFiManager`, and I'm afraid I can't do anything more to add.

Good Luck,

**Reply by deruiter** [reply]
Thank you so much for your reply! BTW: I changed the subject because your code is excellent and for ME it is a bit complicated. The new subject title reflects my lack of knowledge better instead of giving your amazing library the blame. ;)

I made some great projects (hackster.io/edr1924), some of which took me a year of research and learning so I am used to dig. So I have to learn a lot more this time.

IT DOES WORK now using the Arduino IDE 1.8.16 (which I don't like) so I now have to figure out how to get it to work on PlatformIO.

Thank you!

**Reply by khoih-prog** [reply]
I'm glad it's working for you now. BTW, I just have a look at your amazing https://www.hackster.io/edr1924. It's so professional and helpful.

**Reply by deruiter** [reply]
Thank you!  :)
